torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2677,14 +2681,16 @@ def sample_inputs_elementwise_unary(
     if not op_kwargs:
         op_kwargs = {}
 
+    _L = S if kwargs.get("small_inputs_only", False) else L
```

Comment:
nit: move with the others above

Reply:
Not sure if possible - they are supposed to apply for different sample inputs functions